apps/user/libs/utils/swagger.ts

```diff
@@ -0,0 +1,13 @@
+import { INestApplication } from '@nestjs/common';
+import { DocumentBuilder, SwaggerModule } from '@nestjs/swagger';
+
+export function setupSwagger(app: INestApplication): void {
```

Comment:
Awesome you are using swagger, can you give the guide to access it? In the Readme?